doc/networking.md

```diff
@@ -427,6 +428,18 @@ NETCONF or using the CLI:
     192.168.200.0/24              192.168.1.1                     20  static
     admin@example:/>
 
+#### OSPFv2 Routing
+Remember to enable [IPv4 forwarding](#IPv4-forwarding) for the
+interfaces you want to run OSPFv2.
+
+    admin@example:/config/> edit routing control-plane-protocol ietf-ospf:ospfv2 name default
+    admin@example:/config/routing/control-plane-protocol/ietf-ospf:ospfv2/name/default/> set ospf area 0.0.0.0 interface e0 enabled true
+    admin@example:/config/routing/control-plane-protocol/static/name/default/> leave
+    admin@example:/>
+
+> **Note:** The only name allowed for a control-plane-protocol is currently
+> *default*.  Meaning, you can only have one instance per routing protocol.
+
```

Comment:
I had this idea that we could add some kind of inference of the name, similar to how we guess interface type based on their names (br0/veth0), and then deviate this away?

Reply:
I have tried that. 
Name is set to key in the list, and i didn't to deviate it then.

Maybe i should give it a try again. I add it to the leftovertask